packages/code-du-travail-frontend/src/common/Breadcrumbs.js

```diff
@@ -53,6 +53,9 @@ const { breakpoints, fonts, spacings } = theme;
 
 const Nav = styled.nav`
   margin-bottom: ${spacings.small};
+  @media print {
```

Comment:
I find it could be useful to have the breadcrumbs printed

Reply:
I think breadcrumbs has a navigation purpose mostly. There already is the main theme displayed, in the printed document, just above the title